pkg/api/config.go

```diff
@@ -67,6 +68,7 @@ type Config struct {
 	RegistryStorageAccount     string
 	RegistryConsoleOAuthSecret string
 	RouterStatsPassword        string
+	LoggingWorkspace           string
```

Comment:
Unclear what this means just by seeing it here, mind adding godoc (which we should do for all fields but the rest can be done in another PR)?